tests/validation/tests/v3_api/common.py

```diff
@@ -357,18 +358,24 @@ def assign_members_to_project(client, user, project, role_template_id):
 
 
 def change_member_role_in_cluster(client, user, crtb, role_template_id):
-    crtb = client.update(
-        crtb,
+    client.delete(crtb)
+    crtb = client.create_cluster_role_template_binding(
+        clusterId=crtb.clusterId,
         roleTemplateId=role_template_id,
-        userId=user.id)
+        subjectKind="User",
+        userId=user.id
+    )
     return crtb
```

Comment:
Is there even a point of this test if we have to delete the crtb? It's not testing the same thing anymore, we are basically invalidating the case it intended to test.

Reply:
These aren't tests themselves. They're in `common.py` which is used by other files, e.g. `test_rbac.py`

```
$ grep "change_member_role_in_cluster" tests/validation/tests/v3_api
tests/validation/tests/v3_api/common.py:360:def change_member_role_in_cluster(client, user, crtb, role_template_id):
tests/validation/tests/v3_api/common.py:371:def change_member_role_in_project(client, user, prtb, role_template_id):
tests/validation/tests/v3_api/test_rbac.py:9:    change_member_role_in_cluster,
tests/validation/tests/v3_api/test_rbac.py:10:    change_member_role_in_project,
tests/validation/tests/v3_api/test_rbac.py:106:    change_member_role_in_cluster(
tests/validation/tests/v3_api/test_rbac.py:125:    change_member_role_in_cluster(
tests/validation/tests/v3_api/test_rbac.py:147:    change_member_role_in_project(
tests/validation/tests/v3_api/test_rbac.py:170:    change_member_role_in_project(
```